src/app/util/af-types.h

```diff
@@ -221,6 +223,16 @@ struct EmberAfDefinedEndpoint
      */
     chip::DataVersion * dataVersions = nullptr;
 
+#if CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT > 0
+    /**
+     * Pointer to the memory block to be used for automatic attribute storage if
+     * this is a dynamic endpoint. If set, the memory block pointed at
+     * must have a size equal to endpointType->endpointSize (which is
+     * the sum of all endpointType->clusters[*].clustersize).
+     */
+    uint8_t * dynamicAttributeStorage = nullptr;
```

Comment:
Could we have this a Span instead so it contains it size? That seems easier to check/enforce than having the comment explaining the exact size.

Reply:
makes perfect sense, done